Gemfile

```diff
@@ -39,6 +39,8 @@ gem 'jquery-rails'
 gem 'devise'
 gem 'omniauth-github'
 gem 'github_api'
+gem 'delayed_job_active_record'
+gem 'daemons'
```

Comment:
Do you actually need it in production? I would suggest moving it into `:development, :test`.
Or even better, just use [foreman](https://github.com/ddollar/foreman) to launch all configurations from the same `Procfile`